themes/bootstrap3/js/common.js

```diff
@@ -163,7 +174,7 @@ var VuFind = (function VuFind() {
           var sliceEnd = sliceStart;
           var value = _attrs[attr];
           var regex = new RegExp(` ${attr}=(['"])([^\\1]+?)\\1`);
-          var existing = html.match(regex);
+          var existing = mod.match(regex);
```

Comment:
Oops, another one. Now fixed, and html moved lower. Would probably be better to use `let` as well instead of `var`, but that's another change for another time.